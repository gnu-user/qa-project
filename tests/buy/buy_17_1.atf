The_Godfather_______administrator___100_010.00
The_Godfather_II____administrator___100_010.00
The_Godfather_III___seller__________100_010.00
END_________________________________000_000.00