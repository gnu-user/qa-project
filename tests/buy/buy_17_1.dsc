TITLE
    Ticket Purchase from Seller (Seller Not Selling Ticket)
DESCRIPTION
    Run the login transaction using an admin account. Initiate
    the buy transaction. When prompted, enter a title which 
    exists but is not being sold by one specific seller. When
    prompted, enter 4 tickets to purchase. When prompted, enter
    a seller which exists, but is not selling the ticket 
    specified. Verify that the seller is not accepted, and an 
    error occurs.
DEPENDENCIES
    buy_16
