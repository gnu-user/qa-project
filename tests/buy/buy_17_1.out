Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the event title:
Enter the number of tickets to purchase: 
Enter the seller's username: 
Tickets not found, specified seller is not selling tickets to that event.